### DISCUSSION TITLE
My result using SmallBASIC

### DISCUSSION THREAD

**Discussion by LordSamwi** [PlummersSoftwareLLC/Primes#470] [General]
Using the non MS SmallBASIC on my android tablet. My result is probably at the bottom of the list. 1 loop takes 6.9 seconds.

This basic is an interpreter implementation. Does not support procedures. Unless you set them up as multi-line functions of subroutines. Also does not support any variable typing. It is either a long float or a string. So I was not able to setup a bit array. Figured a work around by taking a number and BIT AND'ing a bit mask value to clear the specific bit.

I do have a question, Is there a reason APL is not in the competition?

if you want to reply to this, don't reply here, send it directly to my samwignv@gmail.com account.

For a giggle here is a partial list of the languages I have used over the past 40+ years:
   6502 machine code; assembler for z80, X86, 6502, ibm370; 
   APL; BASIC (various dialects); C; Objective C; COBOL; FORTRAN; LOGO; PASCAL; RPG II
Of the 14, half were self taught

Good day and keep the chocolate and caffeine coming

Michael Stanley

**Comment by rbergen** [comment]
> I do have a question, Is there a reason APL is not in the competition?

Yes. No one opened a PR to add it. I'd like to add to that, that anybody who might want to do so, still can. 

> if you want to reply to this, don't reply here, send it directly to my &lt;email address&gt; account.

That's generally not how issues (or discussions), work :)
